app/lib/actions/middleware/load_setting_values.rb

```diff
@@ -0,0 +1,36 @@
+module Actions
+  module Middleware
+    class LoadSettingValues < ::Dynflow::Middleware
+      # ::Actions::Middleware::LoadSettingValues
+      #
+      # A middleware to ensure we load current setting values
+
+      def delay(*args)
```

Comment:
What about keyword argument (`**args`)?

Reply:
Dynflow doesn't really expect keyword arguments in there https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/middleware.rb#L28

Reply:
there are non on the parent, are those expected to be added?